src/components/Episode/styles.scss

```diff
@@ -1,7 +1,31 @@
 .wrapper {
-	font-size: 1em; //placeholder styling
-
+	margin-top: 20vh;
+	position: absolute;
+	
 	&[aria-hidden="true"]{
 		opacity: 0;
 	}
-}
\ No newline at end of file
+}
+
+	.content {
+		padding: 10%; 
```

Comment:
is this actually `%` or is it `em`?

Reply:
Did some more calculations and actually the best value (that applies pretty closely over mobile and desktop) is 10vw.